setup.py

```diff
@@ -55,15 +55,25 @@ def get_version():
         for line in lines:
             if line.startswith('Version:'):
                 return line[8:].strip()
-    if os.path.exists("./tools"):
-        print("Running tools/get_version.sh")
-        version = subprocess.getoutput("bash -c 'cd tools; ./get_version.sh'")
+    if os.path.exists("./include/vrv"):
+        with open("./include/vrv/vrvdef.h") as header_file:
+            defines = {}
+            for line in header_file.readlines():
+                if line.startswith("#define"):
+                    line.rstrip()
+                    definition = line[8:].split()
+                    try:
+                        defines[definition[0]] = definition[1]
+                    except IndexError:
+                        pass
+            version = '.'.join(
+                (defines['VERSION_MAJOR'], defines['VERSION_MINOR'], defines['VERSION_REVISION']))
+            if defines['VERSION_DEV'] == 'true':
+                version += '.dev'
     if version.endswith(".dev"):
-        init_sha = subprocess.getoutput(
-            "git log -n 1 --pretty=format:%H -- bindings/python/.pypi-version")
```

Comment:
The `.devXXX` version number in the python wheels is currently based on the changes made since this file was changed for the last time. This means we can reset it when we make a release. Is this feature preserved?

Reply:
Here we count commits in comparison to master. So as soon there is a release (from master branch) it is reset. [GitPython](https://gitpython.readthedocs.io/) makes this very easy.

Reply:
That does not work. If for some reason there is a commit to master (e.g., a change to the README file), it will reset it and that can create conflict on PyPi.